### PR TITLE
Focus basic.html on combined fields and add new U.S. style split-test

### PR DESCRIPTION
**Pull request by mnoorenberghe**
I made basic.html more basic by only including the minimal fields that would be needed (e.g. combined named, combined address field) and simplified the markup.

I added a U.S. style form with separate name, address and phone number fields.

### CODE REVIEW

**Comment by mnoorenberghe** [comment]
To clarify a reason for splitting… I was worried it would be confusing for Nightly users who are linked to basic.html to see duplicate fields for the same type.

File: autocomplete-us-separate-fields.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    <label>Telephone: <input autocomplete="tel-country-code" size="2"><input autocomplete="tel-area-code" size="3" maxlength="3"><input autocomplete="tel-local-prefix" size="3" maxlength="3"><input autocomplete="tel-local-suffix" size="4" maxlength="4">
+    </label>
+    <label>Email: <input autocomplete="email"></label>
+    <p>
+      <input type="submit" value="Submit">
+      <button type="reset">Reset</button>
+    </p>
+  </form>
+
+  <p><button id="btn">Add new input</button></p>
```
Comment (luke-chang) [inline autocomplete-us-separate-fields.html]: I guess this section is not necessary because it was created for testing `DOMInputAdded`.

**Review by luke-chang — COMMENTED** [review]
It's nice to make `basic.html` basic. However, I would prefer to have another demo page containing all supported @autocomplete so we can manually test that all fields are handled well.

File: autocomplete-us-separate-fields.html (hunk 1 above)
Comment (mnoorenberghe) [reply autocomplete-us-separate-fields.html]: I agree we don't need it for now since it doesn't have @autocomplete

**Pull request merged**